Update person.markdown (#21735)
home-assistant/home-assistant.github.io: source/_integrations/person.markdown

**Hint**: When you use multiple device trackers together, especially stationary and GPS trackers, it's advisable to set `consider_home` for stationary trackers as low as possible see [device_tracker](/integrations/device_tracker/#configuring-a-device_tracker-platform)).

> You can manage persons via the UI [![Open your Home Assistant instance and show your people.](https://my.home-assistant.io/badges/people.svg)](https://my.home-assistant.io/redirect/people/) from the person page inside the configuration panel or via `YAML` in your `configuration.yaml` file.

Reviewer: ```suggestion
You can manage persons {% my people title="via the UI from the person page inside the configuration panel" %}  or via `YAML` in your `configuration.yaml` file.
```
